09-PowershellModule/Readme.md

```diff
@@ -0,0 +1,128 @@
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId> -TenantId <TenantId>
+```
+The parameters used are:
+- SubscriptionId – Subscription ID is the identifier of your Azure subscription
+- TenantId - Tenant ID is the identifier for your Azure tenant 
+
+You need to have at least **Reader** role at the subscription scope to run this command. 
+
+### Security scanner for azure - How to fix findings?
+
+AzTS cmdlet generate outputs which are organized as under: 
+- summary information of the control evaluation (pass/fail) status in a CSV file
+
+To address findings, you should do the following:
+1. See the summary of control evaluation first in the CSV file. (Open the CSV in XLS. Use "Format as Table", "Hide Columns", "Filter", etc.)
+2. Review controls that are marked as "Failed" or "Verify"
+3. The 'Remediation Steps' column for each control in the XLS will tell you the command/steps needed to resolve the issue.
+
+
+### Target specific controls during security scan for azure
+
+The security scanner for Azure supports multiple parameters as specified below:
+- SubscriptionId – Subscription ID is the identifier of your Azure subscription 
+- TenantId - Tenant ID is the identifier of your Azure subscription
+- FilterTags  - Comma separated tags to filter the security controls. e.g.: RBAC, Automated, etc.
+- ControlIds  - Comma separated control ids to filter the security controls. e.g.:Azure_Storage_AuthN_Dont_Allow_Anonymous,"Azure_APIManagement_DP_Use_HTTPS_URL_Scheme, etc.
+- ResourceTypeNames - Comma separated resource type to filter the security controls. e.g.:Storage,SubsciptionCore, etc.
+- ExcludeControlIds - Comma separated control ids to exclude the security controls. e.g.:Azure_Storage_AuthN_Dont_Allow_Anonymous,Azure_APIManagement_DP_Use_HTTPS_URL_Scheme, etc.
+- ExcludeResourceTypeNames - Comma separated resource type to exclude the security controls. e.g.:Storage,SubsciptionCore, etc.
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId> -TenantId <TenantId> [-ControlIds <ControlIds>] [-FilterTags <FilterTags>] [-ResourceTypeNames <ResourceTypeNames>] [-ExcludeControlIds <ExcludeControlIds>] [-ExcludeResourceTypeNames <ExcludeResourceTypeNames>] 
+```
+These different parameters would enable you to execute different 'flavors' of secuirty scanner for azure.  
+Here are some examples:
+
+1. Execute only RBAC related controls.
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId> -TenantId <TenantId> -FilterTags "RBAC"
+``` 
+2. Execute all the controls related to storage resource type.
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId>  -TenantId <TenantId> ResourceTypeNames "Storage"
+``` 
+3. Execute specific control related to storage resource type. 
+```PowerShell
+Invoke-AzureScan -SubscriptionId <SubscriptionId> -TenantId <TenantId> -ControlIds Azure_Storage_AuthN_Dont_Allow_Anonymous
+``` 
+4. Exclude specific control related to storage resource type. 
```

Comment:
We are saying 'Exclude' but that parameter is not there.

Reply:
Done